ui/src/pages/publications/[id]/versions/[versionId].tsx

```diff
@@ -190,7 +190,7 @@ const Publication: Types.NextPage<Props> = (props): React.ReactElement => {
 
     const user = Stores.useAuthStore((state: Types.AuthStoreType) => state.user);
     const isBookmarkButtonVisible = useMemo(() => {
-        if (!user || !publicationVersion) {
+        if (!user || !publicationVersion || (publicationVersion && publicationVersion.currentStatus !== 'LIVE')) {
```

Comment:
Please add the same condition as above `useMemo(() => `user && publicationVersion?.currentStatus === 'LIVE'`